src/include/parser/transaction_statement.h

```diff
@@ -33,8 +33,34 @@ class TransactionStatement : public SQLStatement {
   TransactionStatement(CommandType type)
       : SQLStatement(StatementType::TRANSACTION), type(type) {}
 
-  virtual void Accept(SqlNodeVisitor* v) override {
-    v->Visit(this);
+  virtual void Accept(SqlNodeVisitor *v) override { v->Visit(this); }
+
+  const std::string GetInfo(int num_indent) const {
+    std::ostringstream os;
+    os << StringUtil::Indent(num_indent) << "TransactionStatement\n";
+    os << StringUtil::Indent(num_indent + 1) << "Type: ";
+    switch (type) {
+      case kBegin:
+        os << "Begin\n";
```

Comment:
add new line outside of the `switch` statement

Reply:
Done